### ISSUE TITLE
Ubuntu 18.04 install warning for brave-keyring

### ISSUE THREAD

**Issue by celeduc** [brave/brave-browser#4479]
## Description 
When installing the Brave browser on Ubuntu 18.04 according to the [instructions](https://brave-browser.readthedocs.io/en/latest/installing-brave.html#linux), a warning is produced for brave-keyring: "The postinst maintainerscript of the package brave-keyring seems to use apt-key (provided by apt) without depending on gnupg or gnupg2. This will BREAK in the future and should be fixed by the package maintainer(s). Check first if apt-key functionality is needed at all - it probably isn't!"

## Steps to Reproduce

   1. Install Ubuntu 18.04
   2. Follow instructions on https://brave-browser.readthedocs.io/en/latest/installing-brave.html#linux
   3. Observe error 

## Actual result:
```
...
Setting up brave-keyring (1.2) ...
Warning: The postinst maintainerscript of the package brave-keyring
Warning: seems to use apt-key (provided by apt) without depending on gnupg or gnupg2.
Warning: This will BREAK in the future and should be fixed by the package maintainer(s).
Note: Check first if apt-key functionality is needed at all - it probably isn't!
OK
...
```

## Expected result:
No warnings

## Reproduces how often: 
Always

## Brave version (brave://version info)

Brave | 0.64.76 Chromium: 74.0.3729.157 (Official Build) (64-bit)
-- | --
Revision | 7b16107ab85c5364cdcd0b2dea2539a1f2dc327a-refs/branch-heads/3729@{#998}
OS | Linux

**Comment by rebron** [comment]
cc: @mbacchi Can you take a look when you get a chance?

**Comment by nylag** [comment]
Confirmed, same issue.
Followed: [https://brave-browser.readthedocs.io/en/latest/installing-brave.html#ubuntu-16-04-and-mint-18](url) instructions on bionic and had same issue. Instructions were:

> curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -
>
>source /etc/os-release
>
>echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ $UBUNTU_CODENAME main" | sudo tee /etc/apt/sources.list.d/brave-browser-release-${UBUNTU_CODENAME}.list
>
>sudo apt update
>
>sudo apt install brave-keyring brave-browser

**-******************************************-**
Full response follows (PS: this is a very clean fresh install from full format with very few hours of use)
**-******************************************-**
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  linux-headers-4.18.0-18 linux-headers-4.18.0-18-generic
  linux-image-4.18.0-18-generic linux-modules-4.18.0-18-generic
  linux-modules-extra-4.18.0-18-generic
Use 'sudo apt autoremove' to remove them.
The following NEW packages will be installed:
  brave-browser brave-keyring
0 upgraded, 2 newly installed, 0 to remove and 70 not upgraded.
Need to get 75.0 MB of archives.
After this operation, 246 MB of additional disk space will be used.
Get:1 https://brave-browser-apt-release.s3.brave.com bionic/main amd64 brave-browser amd64 0.64.77 [75.0 MB]
Get:2 https://brave-browser-apt-release.s3.brave.com bionic/main amd64 brave-keyring all 1.2 [3,336 B]
Fetched 75.0 MB in 34s (2,202 kB/s)                                            
Selecting previously unselected package brave-browser.
(Reading database ... 199496 files and directories currently installed.)
Preparing to unpack .../brave-browser_0.64.77_amd64.deb ...
Unpacking brave-browser (0.64.77) ...
Selecting previously unselected package brave-keyring.
Preparing to unpack .../brave-keyring_1.2_all.deb ...
Unpacking brave-keyring (1.2) ...
Setting up brave-browser (0.64.77) ...
update-alternatives: using /usr/bin/brave-browser to provide /usr/bin/x-www-browser (x-www-browser) in auto mode
update-alternatives: using /usr/bin/brave-browser to provide /usr/bin/gnome-www-browser (gnome-www-browser) in auto mode
Setting up brave-keyring (1.2) ...
Warning: The postinst maintainerscript of the package brave-keyring
Warning: seems to use apt-key (provided by apt) without depending on gnupg or gnupg2.
Warning: This will BREAK in the future and should be fixed by the package maintainer(s).
Note: Check first if apt-key functionality is needed at all - it probably isn't!
OK
Processing triggers for mime-support (3.60ubuntu1) ...
Processing triggers for desktop-file-utils (0.23-1ubuntu3.18.04.2) ...
Processing triggers for man-db (2.8.3-2ubuntu0.1) ...
Processing triggers for gnome-menus (3.13.3-11ubuntu1.1) ...

**Comment by rebron** [comment]
cc: @fmarier can you take a look?

**Comment by fmarier** [comment]
That warning is also tracked here: https://github.com/brave/brave-release/issues/4

**Comment by fmarier** [comment]
This should now be fixed in `brave-keyring` 1.3 which we just released.
